plugins/exporter.koplugin/main.lua

```diff
@@ -104,6 +104,8 @@ local Exporter = WidgetContainer:extend{
         markdown = require("target/markdown"),
         readwise = require("target/readwise"),
         text = require("target/text"),
+        MyClippings = require("target/myClippings"),
```

Comment:
nit: we don't do camelCase ;).

Reply:
Refactored, thanks for pointing it out :)